strings-csharp/DeterminingIfStringIsNumericOnly/DeterminingIfStringIsNumericOnly/IsStringNumber.cs

```diff
@@ -0,0 +1,49 @@
+using System.Text.RegularExpressions;
+using BenchmarkDotNet.Attributes;
+
+namespace DeterminingIfStringIsNumericOnly;
+
```

Comment:
It would be nice to add the summaryorderer here so that the benchmarks are ordered either slowest to fastest or fastest to slowest

Reply:
Added